test/test_autograd.py

```diff
@@ -871,7 +871,10 @@ def test_no_unnecessary_save(self):
         for i in range(3):
             x.detach_()
             x.copy_(mu + i)
-            loss += (x * torch.tensor([float(i)])).sum()
+            ft = torch.tensor([float(i)])
```

Comment:
did you mean to leave this in?  This looks like some debugging leftovers.

Reply:
You're right it's leftovers, but I left it thinking that if anyone else has a failing test in this code, they might also be helped by a more specific line number in the error message.

Reply:
sure, if you want to leave it, seems fine.